plugins/liferay-maven-plugin/src/test/resources/projects/servicebuilder/builder/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>it</groupId>
+	<artifactId>builder</artifactId>
+	<packaging>pom</packaging>
+	<name>builder</name>
+	<version>1.0</version>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>com.liferay.maven.plugins</groupId>
+				<artifactId>liferay-maven-plugin</artifactId>
+				<version>${liferay.maven.plugin.version}</version>
+				<configuration>
+					<autoDeployDir>target/deploy</autoDeployDir>
```

Comment:
this test should just be about testing the generation of service builder artifacts, not about deployment